java/code/src/com/suse/manager/api/HttpApiResponse.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 SUSE LLC
+ * Copyright (c) 2023 SUSE LLC
```

Comment:
Do not forget to revert this before merging.

Reply:
Yup.